cmd/src/gateway_benchmark.go

```diff
@@ -55,26 +58,49 @@ Examples:
 			return cmderrors.Usage("additional arguments not allowed")
 		}
 
-		// Create HTTP client with TLS skip verify
-		client := &http.Client{Transport: &http.Transport{}}
-
-		endpoints := map[string]string{
-			"HTTP":                fmt.Sprintf("%s/gateway", cfg.Endpoint),
-			"HTTP then WebSocket": fmt.Sprintf("%s/gateway/http-then-websocket", cfg.Endpoint),
+		var (
+			gatewayWebsocket, sourcegraphWebsocket *websocket.Conn
+			err                                    error
+			httpClient                             = &http.Client{}
+			endpoints                              = map[string]any{}
```

Comment:
I think this would help clarify that `any`.

```suggestion
			endpoints                              = map[string]any{} // Values: URL `string`s or `*websocket.Conn`s
```